DOCS-7870-made edits (#63)
mulesoft/docs-service-mesh: modules/ROOT/pages/configure-service-mesh.adoc

* xref::configure-using-CLI.adoc[Using CLI]
* xref::configure-using-CRD.adoc[Using CRD]

== Configuring Response Headers for Anypoint Service Mesh

Services protected by Anypoint Service Mesh return extra headers. These headers are added by Envoy and the gRPC adapter.

> 

Reviewer: Remove extra linespace?

Author: Yes, Done! Thanks!

> You can remove these headers to avoid returning them to the originating client.

Reviewer: Should we say something about why or under what conditions the audience wants or needs to do this?

Author: It is configured if the users don't want extra headed to be returned to the originating API. I think that comes across clearly. Done! Thanks!

To remove the headers:

. Log in to your Ingress server.

> . Search and remove the following headers added by Envoy:

Reviewer: ```suggestion
. Search for and remove the following headers added by Envoy:
```

Author: Done! Thanks!

> * x-envoy-upstream-service-time

Reviewer: ```suggestion
* `x-envoy-upstream-service-time`
```

Author: Done! Thanks!

> * x-envoy-decorator-operation

Reviewer: ```suggestion
* `x-envoy-decorator-operation`
```

Author: Done! Thanks!

> . Search and remove the following headers added by the gRPC adapter:

Reviewer: ```suggestion
. Search for and remove the `x-mule-report` added by the gRPC adapter.
```

Author: Done! Thanks!

> * x-mule-report

Reviewer: Omit single-item bullets.
```suggestion
```

Author: Done! Thanks!
